src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,3 @@
+$cell-size: 100px;
+$gap-size: 1px;
+$padding: 5px;
```

Comment:
Consider using more descriptive variable names, like $calendar-padding, for better readability.